docs/reference/search/search-shards.asciidoc

```diff
@@ -166,3 +166,7 @@ routing values have been specified.
     A boolean value whether to read the cluster state locally in order to
     determine where shards are allocated instead of using the Master node's
     cluster state.
```

Comment:
I think that we need to adjust a sentence above where we say that if filtered aliases are used, they will be returned.